readme.md

```diff
@@ -47,11 +47,11 @@ via a web browser. See [security warning](https://nette.org/security-warning).**
 
 Notice: Composer PHP version
 ----------------------------
-This project forces `PHP 5.6` as your PHP version for Composer packages. If you have newer version on production you should change it in `composer.json`.
+This project forces `PHP 7.1` as your PHP version for Composer packages. If you have newer version on production you should change it in `composer.json`.
 ```json
 "config": {
 	"platform": {
-		"php": "7.0"
+		"php": "7.1"
```

Comment:
Sorry, 7.2 was right! 

or `"php": "7.2"   ---> changed to 7.2`

Reply:
I haven't think it though, `7.2` is right. 👍

Reply:
fixed